dashboard/src/components/Cards/IssuesList.tsx

```diff
@@ -13,26 +15,32 @@ import type { TIssue } from '@/types/general';
 
 interface IIssuesList {
   issues: TIssue[];
+  failedWithUnknownIssues?: number;
   title: IBaseCard['title'];
 }
 
-const IssuesList = ({ issues, title }: IIssuesList): JSX.Element => {
+const IssuesList = ({
+  issues,
+  failedWithUnknownIssues,
+  title,
+}: IIssuesList): JSX.Element => {
+  const intl = useIntl();
   const hasIssue = issues.length > 0;
 
   const titleElement = (
     <span>
       {title}
-      {hasIssue && (
+      {(hasIssue || failedWithUnknownIssues) && (
         <ColoredCircle
           className="ml-2 font-normal"
           backgroundClassName={ItemType.Error}
-          quantity={issues.length}
+          quantity={issues.length + (failedWithUnknownIssues ? 1 : 0)}
         />
       )}
     </span>
   );
 
-  const contentElement = !hasIssue ? (
+  const contentElement = !failedWithUnknownIssues ? (
```

Comment:
dont change this just make` const hasIssue = issues.length > 0 || failedWithUnknownIssues > 0;`

Reply:
In this case, `const hasIssue = issues.length > 0` could also just be `const hasIssue = issues.length`, is this change acceptable or does it just make the code less readable?

Reply:
After more thought I don't think I could do just `issues,length` because that could show a random 0 instead of my component